modules/client-configuration/pages/supported-features.adoc

```diff
@@ -210,7 +210,7 @@ ifeval::[{uyuni-content} == true]
 | {check}
 
 | {almalinux} 9
-| {x86_64}, {ppc64le}, {ibmz}, {aarch64}
+| {x86_64}, {aarch64}
```

Comment:
Not right. For Alma9 we support  x86_64, aarch64, ppc64le, s390x, at least on Uyuni :-)